apps/cowswap-frontend/src/modules/tokensList/pure/TokensVirtualList/index.tsx

```diff
@@ -31,6 +32,9 @@ export function TokensVirtualList(props: TokensVirtualListProps) {
   const { allTokens, selectedToken, balancesState, onSelectToken, unsupportedTokens, permitCompatibleTokens } = props
   const { values: balances, isLoading: balancesLoading } = balancesState
 
+  const { account } = useWalletInfo()
```

Comment:
This is a pure component, it's not supposed to contain external dependencies.
The account should be passed through props

Reply:
@shoom3301 I have addressed your comment now